app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,44 @@
+<h2 class="headline">Sign up</h2>
+
+<%= stylesheet_link_tag 'application', media: 'all' %>
+
+<div class="flash-messages">
+  <% flash.each do |key, value| %>
+    <div class="alert alert-<%= key %>">
+      <%= value %>
+    </div>
+  <% end %>
+</div>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name), id: "form-container") do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <% f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, class: "form-control", width: "100px", placeholder: "Name:" %>
+  </div>
+  <br />
+  <div class="field">
+    <% f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", class: "form-control", width: "100px", placeholder: "Email:" %>
+  </div>
+  <br />
+  <div class="field">
+    <% f.label :password %>
+    <% if @minimum_password_length %><br />
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password", class: "form-control", width: "100px", placeholder: "Password:" %>
+  </div>
+
+  <div class="field">
+    <% f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password", class: "form-control", width: "100px", placeholder: "Password confirm:" %>
+  </div>
+  <br />
+  <div class="actions">
+    <%= f.submit "Sign up", class: "submit-btn2" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] Nice job so far on the authentication pages layout 👌 This activity, however, requires the modification of the views in _devise_ for the registration also matches the styling of the [provided wireframe(s)](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md). Kindly ensure that the form (inputs and buttons) stretches across the screen and is centered accordingly for the **signup/login** page, as seen in the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md):

**Your Signup/login view:**      |  **Wireframe Signup/login view (Expected):**
:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://github.com/jcmunav63/my-ror-blog/assets/12745474/371dbb3b-1b17-4294-8d90-06c208406618"> | <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/192667669-04020765-a648-4c50-ab45-7d68eda203e7.png">